ncard_app/templates/tables/people.html

```diff
@@ -44,6 +47,7 @@ <h1>People</h1>
                     <td class="" data-id="{{ person.id }}" data-type="ncard_relation">{{ person.get_ncard_relation_display }}</td>
                     <td class="" data-id="{{ person.id }}" data-type="project">{{ person.project }}</td>
                     <td class="" data-id="{{ person.id }}" data-type="linkedin">{{ person.linkedin }}</td>
+                    <td class="" data-id="{{ person.id }}" data-type="twietter">{{ person.twitter }}</td>
```

Comment:
This data-type is misspelled. Instead of fixing the spelling, consider removing the attribute if it's unused.